test/load.test.ts

```diff
@@ -62,6 +62,17 @@ const mockedKVs = [{
 }, {
     key: "app5.settings",
     value: "placeholder"
+}, {
+    key: ".appconfig.featureflag/Beta",

```

Comment:
Can we avoid hard coding feature prefix and content type here?

https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/appconfiguration/app-configuration/src/featureFlag.ts#L11

Reply:
I don't think there's much difference.

To be serious, my preference is hard code, as it's a test case. It's clearer to see the raw input and output. And imagine that one day if there is a regression in SDK providing the wrong prefix/contentType, we can detect it.